src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -585,3 +596,241 @@ describe('VAOS <RequestedAppointmentDetailsPage>', () => {
+            .format('YYYY-MM-DDTHH:mm:ss[Z]'),
+        },
+        {
+          start: moment(testDate)
+            .add(4, 'days')
+            .format('YYYY-MM-DDTHH:mm:ss[Z]'),
+        },
+      ],
+      serviceType: '323',
+      start: null,
+      status: 'proposed',
+    };
+
+    mockSingleVAOSRequestFetch({ request: appointment });
+
+    const facility = getVAFacilityMock();
+    facility.attributes = {
+      ...facility.attributes,
+      id: 'vha_442GC',
+      uniqueId: '442GC',
+      name: 'Cheyenne VA Medical Center',
+      address: {
+        physical: {
+          zip: '82001-5356',
+          city: 'Cheyenne',
+          state: 'WY',
+          address1: '2360 East Pershing Boulevard',
+        },
+      },
+      phone: { main: '307-778-7550' },
+    };
+
+    mockFacilityFetch('vha_442GC', facility);
+
+    const screen = renderWithStoreAndRouter(<AppointmentList />, {
+      initialState: initialStateVAOSService,
+      path: `/requests/${appointment.id}`,
+    });
+
+    // Verify page content...
+    await waitFor(() => {
+      expect(document.activeElement).to.have.tagName('h1');
+    });
+
+    expect(
+      screen.getByRole('heading', {
+        level: 1,
+        name: 'Pending primary care appointment',
+      }),
+    );
+
+    expect(screen.baseElement).to.contain('.usa-alert-info');
+    expect(screen.baseElement).to.contain.text(
+      'The time and date of this appointment are still to be determined.',
+    );
+
+    expect(screen.getByText('Cheyenne VA Medical Center')).to.be.ok;
+    expect(screen.baseElement).to.contain.text(
+      'Pending primary care appointment',
+    );
+    expect(screen.baseElement).to.contain.text('VA Appointment');
+    expect(screen.baseElement).to.contain.text('Cheyenne VA Medical Center');
+    expect(screen.baseElement).to.contain.text('2360 East Pershing Boulevard');
+    expect(screen.baseElement).to.contain.text('Cheyenne, WY 82001-5356');
+    expect(screen.baseElement).to.contain.text('Main phone:');
+    expect(screen.baseElement).to.contain.text('307-778-7550');
+    expect(screen.baseElement).to.contain.text('Preferred date and time');
+    expect(screen.baseElement).to.contain.text(
+      `${moment(appointment.attributes.requestedPeriods[0].start).format(
+        'ddd, MMMM D, YYYY',
+      )} in the afternoon`,
+    );
+    expect(screen.baseElement).to.contain.text(
+      `${moment(appointment.attributes.requestedPeriods[1].start).format(
+        'ddd, MMMM D, YYYY',
+      )} in the afternoon`,
+    );
+    expect(screen.baseElement).to.contain.text('Follow-up/Routine');
+
+    expect(await screen.findByText(/A message from the patient/i)).to.be.ok;
+    expect(screen.baseElement).to.contain.text('veteranemailtest@va.gov');
+    expect(screen.baseElement).to.contain.text('212-555-1212');
+    expect(screen.baseElement).to.contain.text('Call morning');
+  });
+
+  it('should render CC request details with a VAOS appointment', async () => {
```

Comment:
We normally would also want to test what happens when the provider fetch fails, since that's a new api call, but I think I'm going to create a ticket for that and ask the designers what they want to happen in that case.